app/components/Settings/Scheduler.react.js

```diff
@@ -0,0 +1,262 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import ReactDataGrid from 'react-data-grid';
+import ms from 'ms';
+import matchSorter from 'match-sorter';
+import Highlight from 'react-highlight';
+
+import { Link } from '../Link.react';
+import { Row, Column } from '../Layout.react';
+import Modal from '../Modal.react';
+import { plotlyUrl } from '../../utils/utils';
+
+const NO_OP = () => {};
+
+export const SQL = (props) => <Highlight className="sql">{props.children}</Highlight>;
+SQL.propTypes = {
+  children: PropTypes.string
+};
+
+class QueryFormatter extends React.Component {
+  static propTypes = {
+    value: PropTypes.object
```

Comment:
Two options here:
- either rename the property so that its meaning doesn't need documenting; e.g. rename `value` to `query`
- or document its meaning using JSDoc

Reply:
It has to remain `value` because that is what `react-data-grid` expects (:  Do you want me to play a full JSDoc block above `static propTypes = {` or just inline comments describing `value`? Only ask because we already know we are labelling props, and I am curious of your preference 👍